### PR TITLE
KEYCLOAK-11647 Update to 7.3.5

### PR DESCRIPTION
**Pull request by ASzc**

### CODE REVIEW

File: image.yaml [hunk 1]
```diff
@@ -59,7 +59,7 @@ modules:
           - name: jboss-eap-modules
             git:
                   url: https://github.com/jboss-container-images/jboss-eap-modules.git
-                  ref: EAP_724_CR2
+                  ref: EAP_724_1
```
Comment (iankko) [inline image.yaml]: Ditto here. It needs to be some _EAP_725_\* tag.

**Review by iankko — CHANGES_REQUESTED** [review]
Needs to wait till _EAP_725_\* tag is available.

**Comment by ASzc** [comment]
@iankko updated

**Review by hmlnarik — APPROVED** [review]

**Comment by hmlnarik** [comment]
Thanks @ASzc

**Pull request merged**